src/html/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no" />
+    <title><%= htmlWebpackPlugin.options.title || 'Webpack App'%></title>
```

Comment:
Let's add something more fancy here. You are good with the names.

Or just "Stable coin converter" for now